File: FoodKing.Services/Database/DailyMenu.cs
using System;
using System.Collections.Generic;

namespace FoodKing.Services.Database;

public partial class DailyMenu
{
    public int Id { get; set; }

    public string Title { get; set; } = null!;
    public DateTime CreatedAt { get; set; } = DateTime.Now;
    public DateTime UpdatedAt { get; set; } = DateTime.Now;
    public bool? SoftDelete { get; set; }

}

File: UI/foodking_admin/lib/models/daily_menu.dart
class DailyMenu {
  final int? id;
  final String? title;
  final String? createdAt;
  final String? updatedAt;

  DailyMenu({this.id, this.title, this.createdAt, this.updatedAt});

  factory DailyMenu.fromJson(Map<String, dynamic> json) {
    return DailyMenu(
        id: json['id'],
        title: json['title'],
        createdAt: json['createdAt'],
        updatedAt: json['updatedAt']);
  }

  Map<String, dynamic> toJson() {
    return {
      'id': id,
      'title': title,
      'createdAt': createdAt,
      'updatedAt': updatedAt
    };
  }
}
